zip-0207.rst

```diff
@@ -434,7 +349,8 @@ Example implementation
                 }
 
                 if (!found) {
-                    return state.DoS(100, error("%s: founders reward missing", __func__), REJECT_INVALID, "cb-no-founders-reward");
+                    return state.DoS(100, error("%s: founders reward missing", __func__),
+                                     REJECT_INVALID, "cb-no-founders-reward");
                 }
```

Comment:
How much more effort would it take to make this code actually runnable with some tests?

Reply:
When we write the implementation PR, we can update the ZIP with whatever code is actually used.